pkg/timelock/scheduler.go

```diff
@@ -159,11 +160,11 @@ func (tw *Worker) dumpOperationStore(now func() time.Time) {
 	tw.logger.Info().Msgf("generating logs with pending operations in %s", logPath+logFile)
 
 	// Get the earliest block from all the operations stored by sorting them.
-	blocks := make([]int, 0)
+	blocks := make([]uint64, 0)
 	for _, op := range tw.store {
-		blocks = append(blocks, int(op[0].Raw.BlockNumber))
+		blocks = append(blocks, op[0].Raw.BlockNumber)
 	}
-	sort.Ints(blocks)
+	slices.Sort(blocks)
```

Comment:
This seems like an unnecessary conversion to an int and can lead to an integer overflow.

This was most likely done so that `sort.Int` could be used, but this can now be replaced with `slices.Sort` which handles multiple int types with generics